precepte-core-cats/src/main/scala/HackSI2712.scala

```diff
@@ -48,25 +48,33 @@ trait PrecepteHackSI2712[TCA, TC[_[_], _], M[_[_]], F[_], Ta, MS, UMS, A] {
 object PrecepteHackSI2712 {
```

Comment:
Is this still needed? Didn't the SI2712 fix get backported to scala 2.11.9?

Reply:
Probably, but i don't want to break code depending on it until i'm sure it is useless.